Add AB test metrics eda

mammykins opened the pull request.

ff-l commented on notebooks/eda/generating_ab_test_data.ipynb:
```diff
@@ -0,0 +1,505 @@
+   "metadata": {},
+   "source": [
+    "\\begin{equation*}\n",
+    "\\frac{total number of journeys including at least one click on a related link}{total number of journeys}\n",
```
If we're in latex land, use \textrm{} or \text{} or something equivalent, so the wordy bits are not a single word.

mammykins replied:
thanks I'm new to it and couldn't find a quick way to do it when googling

mammykins commented:
This is finished, could do with a quick review to confirm I've derived the metrics correctly. 

Could also delete the notebook that doesn't show working as is now redundant @suganya-s given the one I added is succinct version?

We can copy the notebook I added over to our ab_analysis repo. I created a metadata dir in the DATA_DIR to hold the document_type.csv.gz

ff-l commented:
I think some commits need to be squashed as they are way too brief.
